app/views/decidim/proposals/proposals/index.html.erb

```diff
@@ -17,7 +17,7 @@
               </div>
             </div>
             <div class="map-info__button">
-              <a href="${link}" class="button button--sc">
```

Comment:
Hey I think the encoding breaks the template, I tried to add a geocoded proposal and the link on the map is broken 

Here is the link I get
https://localhost:3000/processes/glow-message/f/3/%24%7Blink%7D